package.json

```diff
@@ -355,6 +359,7 @@
     "chromedriver": "^119.0.1",
     "classnames": "2.3.1",
     "compare-versions": "3.5.1",
+    "cypress": "9.5.4",
```

Comment:
Will all of the plugins honor the cypress version here? Or it will be difficult to upgrade Cypress version and as far as I know, latest Cypress version has reached v13. Though this is the version we used in Cypress test repo, we have to add this version to a centralized place for upgrade in the future.

Reply:
I think this is for this first iteration. The plan is to add Functional test repo as a dependency to consume cypress version and its config in the component repositories.

Reply:
Sounds good, please try to keep only one place to store the cypress version and some common configs.

Reply:
The security plugin needs Cypress >= 12 for cross-origin testing. I opened a [META](https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/801) to track the upgrade of Cypress in the FTR repo, but in order to provide adequate testing for cross-origin cases, the security-dashboards-plugin added cypress tests directly in the security-dashboards-plugin repo and uses [Cypress 13](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/package.json#L30).

Can we allow dashboards-plugins to have flexibility with the version of cypress required for their testing without forcing all of them to use a version of cypress that does not support a feature needed for their plugin's testing needs. The reason security-dashboards-plugin needs cross-origin testing is to test single sign on behavior with different authentication providers like SAML and OpenID.